src/main/java/org/jenkinsci/plugins/edithelp/EditHelpRootAction.java

```diff
@@ -0,0 +1,155 @@
+package org.jenkinsci.plugins.edithelp;
+
+import java.io.*;
```

Comment:
It also hard to review diffs with wildcard imports - you can't find where the class come from